### PR TITLE
Refactor prefs.js

### PR DESCRIPTION
**Pull request by lexruee**
Zut, I deleted the wrong branch on my repo. Anyways, this is just a duplicate of https://github.com/LinxGem33/Arc-Menu/pull/120. I need to clean up my Arc-Menu repo :smile:.

-------

These are just simple refactorings that I made with the help of jshint, which is really a nice linter :-). I highly recommend to use it during your programming sessions.
For details of the changes see each commit log.

@LinxGem33  if this pull-request conflicts with your implemented features, we can postpone it to another pull-request, since these are not really important :smile:.

### CODE REVIEW

**Comment by LinxGem33** [comment]
@lexruee 

i will keep this PR on hold just till i finish a couple of the features so i can then test the changes locally for any conflicts after which ill merge with the development branch :smile: , its a good idea by the way incorporating jshint into the development cycle :+1:

**Comment by LinxGem33** [comment]
@lexruee 

decided to merge this PR as i haven’t pushed any code to the development branch yet as i make changes on the fly locally so in theory i could test this PR without any damage to my current development :+1:

File: prefs.js [hunk 1]
```diff
@@ -449,10 +423,10 @@ const AboutPage = new Lang.Class({
...
 
         // Use meta information from metadata.json
-        let releaseVersion = Me.metadata['version'] || 'unknown';
-        let projectName = Me.metadata['name'];
-        let projectDescription = Me.metadata['description'];
-        let projectUrl = Me.metadata['url'];
+        let releaseVersion = Me.metadata.version || 'unknown';
+        let projectName = Me.metadata.name;
+        let projectDescription = Me.metadata.description;
+        let projectUrl = Me.metadata.url;
```
Comment (LinxGem33) [inline prefs.js]: I’m thinking about adding a credits section along with a wiki link.

**Comment by lexruee** [comment]
@LinxGem33 
If you are currently on the dev branch and haven't committed anything, you can easily create a new branch with `git checkout -b feature-xyz` and then commit your changes.

Afterwards, you can rebase your `feature-xyz` branch with the latest dev branch with `git rebase Development-branch`.

File: prefs.js (hunk 1 above)
Comment (lexruee) [inline prefs.js]: **JasonLG1979** did  a nice about page: https://github.com/JasonLG1979/gnome-shell-extensions-mediaplayer/issues/397#issuecomment-320757075

Comment (LinxGem33) [inline prefs.js]: we will need a scroll box element  i.e (new Gtk.ScrolledWindow)

**Comment by LinxGem33** [comment]
@lexruee 
yeah sounds good can always give it a try.

**Pull request merged**